scripts/build/build.sh

```diff
@@ -39,7 +39,13 @@ $REPOROOT/scripts/compile/compile.sh
 
 # Run tests on the stage2 output
 header "Testing stage2..."
+START_TOOLS=$DOTNET_TOOLS
+START_PATH=$PATH
+export DOTNET_TOOLS=$STAGE1_DIR
```

Comment:
This should be wrapped in a method in _common. Like here, it's too easy to mix & max Stage1 Dotnet_HOME and Stage2 bin, etc. Same in PS1